package.json

```diff
@@ -141,7 +152,7 @@
     "shelljs": "^0.8.5",
     "shx": "^0.2.2",
     "ts-node": "^10.4.0",
-    "typescript": "^4.5.2"
+    "typescript": "4.4.4"
```

Comment:
Was this downgrade intentional?

Reply:
Yeah - unfortulately the eslint plugins aren't compatible with the newer version. I suggest a followup PR to get this and the eslint plugins upgraded.

Reply:
OK, I was trying to remember if there was a specific reason I upgraded to 4.5.2. I don't know if there was – so I guess if everything typechecks then this is fine for now